src/uct/cuda/gdr_copy/gdr_copy_ep.c

```diff
@@ -12,6 +12,9 @@
 #include <ucs/sys/math.h>
 #include <ucs/type/class.h>
 
+#if (GDR_API_MAJOR_VERSION < 1 || (GDR_API_MAJOR_VERSION == 1 && GDR_API_MINOR_VERSION < 4))
+#define USE_OLD_GDRCOPY_API 1
```

Comment:
i think it's better to check for gdr_copy_to_mapping function presence by autoconf macro, in https://github.com/openucx/ucx/blob/master/config/m4/gdrcopy.m4